### PR TITLE
Updates Readme.md and Contributing.md

### PR DESCRIPTION
**Pull request by SincerelyBrittany**
## Fixes
Updates Readme.md and Contributing.md to reference how to set up and how to contribute to dev branch.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,40 +16,53 @@ Before you start writing code, make sure there is an issue open. Pull requests w
...
+        * The issue number can be found on the title of the issue you were assigned.
+        * The commit_types are:
+             * (feat): (new feature for the user, not a new feature for build script)
+             * (fix: (bug fix for the user, not a fix to a build script)
+             * (docs): (changes to the documentation)
+             * (style): (formatting, missing semi colons, etc; no production code change)
+             * (refactor): (refactoring production code, eg. renaming a variable)
+             * (test): (adding missing tests, refactoring tests; no production code change)
+             * (chore): (updating grunt tasks etc; no production code change)
+        *  The description/name is the name or title descripting the changes you made.
```
Comment (StashaG) [inline CONTRIBUTING.md:56]: change "descripting" to describing

**Review by StashaG — COMMENTED** [review]
fix typo on line 56

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ If you want to get started contributing code to this project but don't know exac
              * (refactor): (refactoring production code, eg. renaming a variable)
              * (test): (adding missing tests, refactoring tests; no production code change)
              * (chore): (updating grunt tasks etc; no production code change)
-        *  The description/name is the name or title descripting the changes you made.
+        *  The description/name is the name or title description the changes you made.
```
Comment (StashaG) [inline CONTRIBUTING.md:56]: Change "description" to "describing".

**Review by StashaG — APPROVED** [review]
+1 looks good.

**Comment by KiloWattDot** [comment]
+QA/1

**Review by KiloWattDot — APPROVED** [review]
+1/QA

**Review by CatchMeOutsid3 — COMMENTED** [review]
All looks good. Well done.

**Pull request merged**